packages/contracts-bedrock/test/setup/DeployVariations.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Testing utilities
+import { CommonTest } from "test/setup/CommonTest.sol";
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract DeployVariations_Test is CommonTest {
+    function setUp() public override {
+        // prevent calling the base CommonTest.setUp() function
+    }
```

Comment:
Why do we need to prevent it? Would be nice to add that answer into the comment

Reply:
Updated this to: 

```
        // Prevent calling the base CommonTest.setUp() function, as we will run it within the test functions
        // after setting the feature flags
```